src/FixerNameValidator.php

```diff
@@ -27,10 +27,10 @@
      */
     public function isValid($name, $isCustom)
     {
-        if (!$isCustom) {
```

Comment:
what is the change of this function ?

Reply:
Just reversing the condition, so not related to the PR but still a nice change IMO: the "core" naming format is now the default rule, instead of being an exception to it.